doc/.sphinx/_templates/layout.html

```diff
@@ -0,0 +1,7 @@
+{% extends "!layout.html" %}
+
+{% block footer %}
+{{ super() }}
+<div class="footer">YEP THIS IS AN EXAMPLE
```

Comment:
?

Reply:
This doesn't actually show up, but I can delete it anyway